infrastructure/main.tf

```diff
@@ -102,4 +107,19 @@ resource "azurerm_key_vault_secret" "staging_db_database" {
   value        = module.reform-notifications-staging-db.postgresql_database
 }
 
-# endregion
\ No newline at end of file
+# endregion
+
+# region Copy secrets from BulkScan
+
+data "azurerm_key_vault_secret" "bulk_scan_processor_s2s_secret" {
+  key_vault_id = "${data.azurerm_key_vault.bulk_scan_key_vault.id}"
+  name         = "s2s-secret"
```

Comment:
If we copy the secret, it should be test-s2s-secret. But my first suggestion is defining a separate secret pair in s2s, as it's probably very little work.

Reply:
defining lot's of secrets just for tests is not recommended and i did mention that a while back ago when writing those functional tests for the service.

should be re-used. and yes, good notice - i should re-use the test one from bsp

Reply:
I don't know, separate test microservice secrets for tests would make a lot of sense, given this is a separate product